### PR TITLE
Add pyreadline recipe

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by kynan** [comment]
Good to go @conda-forge/core ?

**Comment by jakirkham** [comment]
I think this already [exists]( https://github.com/conda-forge/pyreadline-feedstock ), @kynan.

File: recipes/pyreadline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [not win]
+  number: 0
+  # setup is not using setuptools!
+  script: python setup.py install --record record.txt
```
Comment (jakirkham) [inline recipes/pyreadline/meta.yaml]: FYI the `--record record.txt` is only necessary to go with `--single-version-externally-managed` when using `setuptools`. There is no need to have it on its own.

Comment (kynan) [inline recipes/pyreadline/meta.yaml]: OK, thanks, that's good to know!

**Comment by kynan** [comment]
@jakirkham are you sure this already exists? I couldn't find it. Note that this is *not* the readline library and is completely unrelated to https://github.com/conda-forge/readline-feedstock!

**Comment by jakirkham** [comment]
Pretty sure.

ref: https://github.com/conda-forge/pyreadline-feedstock

**Comment by jakirkham** [comment]
Should add GitHub search through repos apparently doesn't match `.*readline.*` when one types `readline` it only matches `readline.*`.

**Comment by kynan** [comment]
Ah, yes indeed, thanks! I also did a `conda search pyreadline` but I'm on Linux so of course it didn't find any package!

**Comment by jakirkham** [comment]
No problem. This is a semi-regular issue. Even core members have made this mistake both in terms of proposing and merging existing recipes for packages already in conda-forge. There was some work to detect these sorts of issues. It's not totally trivial as package names may not be one-to-one (e.g. some packages have `python-` prefixes), but maybe it is time to give that more attention.

xref: https://github.com/conda-forge/conda-forge.github.io/pull/195

**Pull request closed without merging**